### DISCUSSION TITLE
tlg-catalog/stable/graphs/pharmacokinetic/pkpg01

### DISCUSSION THREAD

**Discussion by giscus** [insightsengineering/tlg-catalog#142] [General]
# tlg-catalog/stable/graphs/pharmacokinetic/pkpg01

Plot of Mean Cumulative Percentage (%) of Recovered Drug in Urine

https://insightsengineering.github.io/tlg-catalog/stable/graphs/pharmacokinetic/pkpg01.html

**Comment by legrasv** [comment]
x_lab does not work in the plot.

**Reply by Melkiades** [reply]
It is fixed by https://github.com/insightsengineering/tern/pull/1126
